### PR TITLE
{lang}[GCCcore/6.4.0] Yasm v1.3.0

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (vanzod) [inline easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-GCCcore-6.4.0.eb]: Change URL with `https://github.com/easybuilders`

File: easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+easyblock = 'ConfigureMake'
+
+name = 'Yasm'
+version = '1.3.0'
+
+homepage = 'http://www.tortall.net/projects/yasm/'
+
+description = """
+ Yasm: Complete rewrite of the NASM assembler with BSD license
+"""
```
Comment (vanzod) [inline easybuild/easyconfigs/y/Yasm/Yasm-1.3.0-GCCcore-6.4.0.eb]: Please make it a single line.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2570.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8ee4d2da3bbd2f02950ed1f86219df22 for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**